<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointManagerDlg</class>
 <widget class="QDialog" name="AppointManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc.qrc">
    <normaloff>:/image/icon/add.png</normaloff>:/image/icon/add.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#AppointManagerDlg
{
	background-image: url(:/image/icon/ui_dlg.png);

	min-width: 450px;
	max-width: 450px;

	min-height: 300px;
	max-height: 300px;
}

QLabel
{
	font-size: 16px;         
	font: &quot;Microsoft YaHei UI&quot;;
	font-weight: bold;
	color: #3dcce1;
}

QPushButton
{
	font-size: 16px;         
	font: &quot;Microsoft YaHei UI&quot;;
	font-weight: bold;
	color: #3dcce1;


	border: none;
	border-radius: 15px;
	
	min-height: 50px;
	max-height: 50px;

	min-width: 130px;
	max-width: 130px;

	background-image: url(:/image/icon/btn_background.png)
}

QPushButton:disabled
{
	background-image: url(:/image/icon/btn_background_disabled.png);
}

QPushButton:hover
{
	background-image: url(:/image/icon/btn_background_hover.png)
}

QPushButton:pressed
{
	background-image: url(:/image/icon/btn_background_pressed.png)
}

QLineEdit
{
	font-size: 14px;
	color: #3dcce1;
	font-weight: bold;
	padding: 1px;
	border-style: solid;
	border: 3px solid #3dcce1;
	border-radius: 5px;

	min-width:150px;  
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>269</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_Ok">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>261</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lb_Name">
        <property name="styleSheet">
         <string notr="true">#lb_Name
{
	min-width: 35px;
	max-width: 35px;

	min-height: 25px;
	max-height: 25px;
}</string>
        </property>
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_Name_Apt">
        <property name="styleSheet">
         <string notr="true">#le_Name_Apt
{
	font-size: 14px;
	padding: 1px;
	border-style: solid;
	border: 1px solid #3dcce1;;
	border-radius: 5px;

	min-width:150px;  
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lb_PhoneNum">
        <property name="styleSheet">
         <string notr="true">#lb_PhoneNum
{
	min-width: 35px;
	max-width: 35px;

	min-height: 25px;
	max-height: 25px;
}</string>
        </property>
        <property name="text">
         <string>电话</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_PhoneNum_Apt">
        <property name="styleSheet">
         <string notr="true">#le_PhoneNum_Apt
{
	font-size: 14px;
	padding: 1px;
	border-style: solid;
	border: 1px solid #3dcce1;;
	border-radius: 5px;

	min-width:150px;  
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lb_Gender">
        <property name="styleSheet">
         <string notr="true">#lb_Gender
{
	min-width: 35px;
	max-width: 35px;

	min-height: 25px;
	max-height: 25px;
}</string>
        </property>
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comb_Gender_Apt">
        <property name="styleSheet">
         <string notr="true">#comb_Gender_Apt
{
	font: &quot;Microsoft YaHei UI&quot;;
	font-size: 14px;
	font-weight: bold;
    color: #3dcce1;

	padding: 1px 15px 1px 3px;
	border:1px solid #3dcce1;
	border-radius:5px 5px 0px 0px;
}

#comb_Gender_Apt QAbstractItemView
{
	outline: none;
	border:3px solid #3dcce1;
}

#comb_Gender_Apt QAbstractItemView::item
{
	color: #3dcce1;
}

#comb_Gender_Apt QAbstractItemView::item::hover
{
	color: #FFFFFF;
	background-color: #3dcce1;
}

#comb_Gender_Apt QAbstractItemView::item::selected
{
	color: #FFFFFF;
	background-color: #3dcce1;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="lb_Age">
        <property name="styleSheet">
         <string notr="true">#lb_Age
{
	min-width: 35px;
	max-width: 35px;

	min-height: 25px;
	max-height: 25px;
}</string>
        </property>
        <property name="text">
         <string>年龄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spb_Age_Apt">
        <property name="styleSheet">
         <string notr="true">#spb_Age_Apt
{
  font: &quot;Microsoft YaHei UI&quot;;
  font-size:14px;
  font-weight: bold;
  color: #3dcce1;

  padding: 1px 15px 1px 3px;
  border:1px solid #3dcce1;
  border-radius:5px 5px 0px 0px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="lb_Time">
      <property name="styleSheet">
       <string notr="true">#lb_Time
{
	min-width: 65px;
	max-width: 65px;

	max-height: 35px;
	max-height: 35px;
}</string>
      </property>
      <property name="text">
       <string>预约时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_Date_Apt">
      <property name="styleSheet">
       <string notr="true">#lb_Date_Apt
{
	font-size: 14px;  
	
	max-height: 35px;
	max-height: 35px;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_Duration_Apt">
      <property name="styleSheet">
       <string notr="true">#lb_Duration_Apt
{
	font-size: 14px;  

	max-height: 35px;
	max-height: 35px;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
